core/arch/arm/kernel/kern.ld.S

```diff
@@ -116,6 +116,12 @@ SECTIONS
 		__exidx_end = .;
 	}
 
+	.ARM.extab : {
+		__extab_start = .;
+		*(.ARM.extab*)
+		__extab_end = .;
+	}
+
```

Comment:
Right. So `*(.ARM.extab*)` is certainly needed, no so sure about `__extab_start`/`__extab_end` but it doesn't hurt much to keep them.